API/Extensions/DependencyInjectionExtensions.cs

```diff
@@ -100,6 +101,8 @@ public static IServiceCollection AddServicesAndRepositories(this IServiceCollect
             services.AddScoped<IWizardPageService, WizardPageService>();
             services.AddScoped<IWizardPageRepository, WizardPageRepository>();
 
+            services.AddScoped<IAssemblyHelper, AssemblyHelper>();
```

Comment:
Why scoped? Is it necessary for each session to get a separate one? If not, why not use a singleton?

Reply:
You're right. I should have chosen singleton. The only reason I initially choose to go for scoped is to support dependency injection for our repositories, but after a closer look, I don't think this is necessary.